config/crd/bases/argoproj.io_argocds.yaml

```diff
@@ -5097,6 +5097,11 @@ spec:
                     required:
                     - enabled
                     type: object
+                  commandArgs:
+                    description: Command arguments for Argo CD server
```

Comment:
The description should maybe say something like `Arguments to be appended to the command arguments for the Argo CD server`